linkis-commons/linkis-common/pom.xml

```diff
@@ -158,6 +158,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>com.ginsberg</groupId>
+            <artifactId>junit5-system-exit</artifactId>
+            <version>1.1.1</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
```

Comment:
junit5-system-exit seems to encapsulate some operations for system exit scenarios, can you explain the main difference from the original way of junit5. 
If using junit5-system-exit is a good way, I suggest adding it as a specification to the [how-to-write-unit-test-code](https://linkis.apache.org/community/how-to-write-unit-test-code) docs

Reply:
junit5-system-exit is a JUnit 5 Extension.It can help us write tests for code that calls `System.exit()`. In the original Jnuit5 way, we asserted that system exits required throwing an exception in `SecurityManager.checkPermission` to interrupt the subsequent logic of System.exit, but thanks to this extension, we were able to simply assert system exits using annotations. BTW, this extension also uses the way of setting `SecurityManager` to achieve its function, which is no different from our original way.